File: app/src/main/java/com/example/learn/testme/Model/Question.java
package com.example.learn.testme.Model;

public class Question {

    private String Question;
    private String AnswerA;
    private String AnswerB;
    private String AnswerC;
    private String AnswerD;
    private String AnswerE;
    private String CorrectAnswer;
    private String IsImageQuestion;

    public String getIsImageQuestion() {
        return IsImageQuestion;
    }

    public void setIsImageQuestion(String isImageQuestion) {
        IsImageQuestion = isImageQuestion;
    }

    public Question(String isImageQuestion) {
        IsImageQuestion = isImageQuestion;
    }

    public String getQuestion() {
        return Question;
    }

    public void setQuestion(String Question) {
        Question = Question;
    }

    public String getAnswerA() {
        return AnswerA;
    }

    public void setAnswerA(String answerA) {
        AnswerA = answerA;
    }

    public String getAnswerB() {
        return AnswerB;
    }

    public void setAnswerB(String answerB) {
        AnswerB = answerB;
    }

    public String getAnswerC() {
        return AnswerC;
    }

    public void setAnswerC(String answerC) {
        AnswerC = answerC;
    }

    public String getAnswerD() {
        return AnswerD;
    }

    public void setAnswerD(String answerD) {
        AnswerD = answerD;
    }

    public String getAnswerE() {
        return AnswerE;
    }

    public void setAnswerE(String AnswerE) {
        AnswerE = AnswerE;
    }

    public String getCorrectAnswer() {
        return CorrectAnswer;
    }

    public void setCorrectAnswer(String correctAnswer) {
        CorrectAnswer = correctAnswer;
    }

    public String getCategoryId() {
        return CategoryId;
    }

    public void setCategoryId(String categoryId) {
        CategoryId = categoryId;
    }

    public String getIsQuestion() {
        return IsQuestion;
    }

    public void setIsQuestion(String isQuestion) {
        IsQuestion = isQuestion;
    }

    private String CategoryId;
    private String IsQuestion;

    public Question(String Question, String answerA, String answerB, String answerC, String answerD, String AnswerE, String correctAnswer, String categoryId, String isQuestion) {
        Question = Question;
        AnswerA = answerA;
        AnswerB = answerB;
        AnswerC = answerC;
        AnswerD = answerD;
        AnswerE = AnswerE;
        CorrectAnswer = correctAnswer;
        CategoryId = categoryId;
        IsQuestion = isQuestion;
    }

    public Question(){

    }

}

File: app/src/main/java/com/example/learn/testme/Model/Question.kt
package com.example.learn.testme.Model

class Question {
    var Question: String? = null
        set(Question) {
            var Question = Question
            Question = Question
        }
    var answerA: String? = null
    var answerB: String? = null
    var answerC: String? = null
    var answerD: String? = null
    var answerE: String? = null
        set(AnswerE) {
            var AnswerE = AnswerE
            AnswerE = AnswerE
        }
    var correctAnswer: String? = null
    var isImageQuestion: String? = null

    constructor(isImageQuestion: String?) {
        this.isImageQuestion = isImageQuestion
    }

    fun getIsQuestion(): String? {
        return IsQuestion
    }

    fun setIsQuestion(isQuestion: String?) {
        IsQuestion = isQuestion
    }

    var categoryId: String? = null
    private var IsQuestion: String? = null

    constructor(Question: String?, answerA: String?, answerB: String?, answerC: String?, answerD: String?, AnswerE: String?, correctAnswer: String?, categoryId: String?, isQuestion: String?) {
        var Question = Question
        var AnswerE = AnswerE
        Question = Question
        this.answerA = answerA
        this.answerB = answerB
        this.answerC = answerC
        this.answerD = answerD
        AnswerE = AnswerE
        this.correctAnswer = correctAnswer
        this.categoryId = categoryId
        IsQuestion = isQuestion
    }

    constructor()
}